modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -195,14 +195,14 @@ pub struct PaymentFee {
 impl PaymentFee {
     pub fn one_percent() -> Self {
         PaymentFee {
-            base: Amount::ZERO,
+            base: Amount::from_sats(50),
```

Comment:
Should this be the same constant as in here? https://github.com/fedimint/fedimint/pull/5600/files

Reply:
No

Reply:
Why not? What is the difference between the two? I thought they were both a minimal for claiming an input

Reply:
The one you linked is only supposed to check that subtracting the fee via saturating_sub did in fact not saturate such that we got the full base fee subtracted.

Reply:
Sorry, I still don't quite understand the relationship between these numbers. i.e if this number was > 100 sats, would that cause problems?

Reply:
If you two need some back and forth to explain and agree, it means me and other devs not all that familiar with it, will have no idea, about some suble but possibly important aspects of this. Please add some comments to give us a chance if we need it in the future.